addon/templates/components/frost-modal-input.hbs

```diff
@@ -1,11 +1,45 @@
-{{#if titleComponent}}
-  {{component titleComponent}}
-{{else}}
-  <div class="input-header">
-    <div class="primary-title">{{title}}</div>
-    {{#if subtitle}}
-      <div class="sub-title">{{subtitle}}</div>
-    {{/if}}
-  </div>
-{{/if}}
-{{yield}}
+{{#ember-remodal
+  buttonClasses='frost-button'
+  forService=true
+  closeOnOutsideClick=false
+  onOpen=(action 'modalOpen')
+  onClose=(action 'modalClose')
+  closeOnEscape=false
+  disableNativeClose=true
+  name=modalName as |modal|}}
+  {{yield}}
+  {{#modal.open}}
+    {{#yield-slot 'openButton'}}
+      {{yield (block-params (hash
+        button=(component 'frost-button')))}}
+    {{/yield-slot}}
+  {{/modal.open}}
+
+  {{#if titleComponent}}
+    {{component titleComponent}}
```

Comment:
We can have the slot default using the `{{else}}` to what you have currently, but yield and let people use the `info-bar` otherwise.  The reason I mention this is that info-bar is getting a few tweaks to align with Marty's requests, so putting these changes behind a component makes sense so that everything remains consistent